src/pages/area-publica/formacao/list/components/dados-destaque/index.tsx

```diff
@@ -65,7 +66,9 @@ const DadosDestaque: React.FC<DadosDestaqueProps> = () => {
       });
     }
   };
-
+  const abrirLinkInscricaoExterna = () => {
+    window.open(linkInscricaoExterna, '_blank');
```

Comment:
Pode tentar usar o component Link

Ex: <Link
            type='link'
            to={acaoFormativaLink}
            target='_blank'
            rel='noreferrer'
            style={{ paddingLeft: '5px' }}
          >
            <span style={{ color: Colors.Suporte.Primary.ERROR }}>* </span>
            {acaoFormativaTexto}
          </Link>